pkg/injector/annotations/annotations_test.go

```diff
@@ -0,0 +1,91 @@
+/*
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package annotations_test
+
+import (
+	"go/ast"
+	"go/parser"
+	"go/token"
+	"reflect"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"golang.org/x/exp/slices"
+
+	"github.com/dapr/dapr/pkg/injector/patcher"
+)
+
+// This test makes sure that the SidecarConfig struct contains all and only the annotations defined as constants in the annotations package.
+func TestAnnotationCompletness(t *testing.T) {
```

Comment:
Reviewers: this test is to make sure that annotations defined in `pkg/injector/annotations` are in-sync with the properties of `SidecarConfig`